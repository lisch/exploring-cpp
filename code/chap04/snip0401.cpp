#include <iostream>
int main() {
#include "snip0401.cc"
}
