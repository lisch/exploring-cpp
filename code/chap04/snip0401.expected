
Circle		?
Triangle	3

Circle		?
Triangle	3

Circle		?
Triangle	3
